`postinstall` script not found

### Description

```bash
➤ YN0009: │ @google/model-viewer@npm:3.2.0 [58ced] couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-dcba39bc/build.log)
➤ YN0007: │ trisetra-decor@workspace:. must be built because it never has been before or the last one failed
➤ YN0000: └ Completed in 6s 348ms
➤ YN0000: Failed with errors in 19s 639ms
```
```bash
# This file contains the result of Yarn building a package (@google/model-viewer@virtual:58ced34e477cff2c62a5e26ca6c375fed90e1af1a4fcc875901614bfd6afa62fbc58ffae3a1f03b621a96e23ef501c7dc5c56da1d8c169f9ff70fe73a6e888d7#npm:3.2.0)
# Script name: postinstall

node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module '/home/subhankar/workspace/trisetra-decor/node_modules/scripts/postinstall.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.16.0

```
This error pops up when I try to install `@google/model-viewer` with Yarn 2+. I works perfectly fine with ```"@google/model-viewer": "^3.1.1"```

### Version
<!--
If you're not sure, paste your script src (like https://unpkg.com/@google/model-viewer) 
into your browser and it will redirect to a numbered version.
-->
- model-viewer: v3.2.0
- yarn: v3.6.1
- node: v18.16.0

### Browser Affected
<!-- Check all that apply and please include the version tested (try chrome://version) -->
- [x] Chrome, version: xx.x.xxxx.xx
- [x] Edge
- [x] Firefox
- [x] IE
- [x] Safari

### OS
<!-- Check all that apply and please include the version tested -->
- [ ] Android
- [ ] iOS
- [x] Linux
- [x] MacOS
- [ ] Windows

--- a/packages/model-viewer/package.json
+++ b/packages/model-viewer/package.json
@@ -47,7 +47,7 @@
   "typings": "lib/model-viewer.d.ts",
   "types": "lib/model-viewer.d.ts",
   "scripts": {
-    "postinstall": "node ../../scripts/postinstall.js",
+    "postinstall": "if [ -f '../../scripts/postinstall.js' ]; then node ../../scripts/postinstall.js; fi",
     "clean": "rm -rf ./lib ./dist",
     "prepare": "if [ ! -L './shared-assets' ]; then ln -s ../shared-assets ./shared-assets; fi && ../shared-assets/scripts/fetch-khronos-gltf-samples.sh",
     "build": "npm run build:tsc && npm run build:rollup",

